src/EditorFeatures/Core/ExternalAccess/VSTypeScript/IVSTypeScriptEditorInlineRenameService.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    internal readonly struct VSTypeScriptInlineRenameLocation
```

Comment:
>internal [](start = 4, length = 8)

We should use -Wrapper pattern for this one (see my other comment on VSTypeScriptInlineRenameReplacementWrapper)